src/components/donation/Steps/Submit/StripeCheckout/Paypal/Paypal.tsx

```diff
@@ -27,23 +22,12 @@ export default function PaypalCheckout(props: PaypalCheckoutStep) {
     splitLiq: liquidSplitPct.toString(),
   });
 
-  const dispatch = useSetter();
-
   return (
-    <Summary
-      classes={{
-        container: "flex flex-col isolate p-4 @md:p-8",
-        split: "mb-auto",
-      }}
-      Amount={currency(details.currency)}
-      amount={+details.amount}
-      splitLiq={liquidSplitPct}
-      onBack={() => dispatch(setStep("summary"))}
-    >
+    <>
```

Comment:
nit: no need for the fragment component, the return can be rewritten to:
```ts
  return (
    isLoading ? (
      <ContentLoader className="rounded h-10 w-40" />
    ) : isError || !orderId ? (
      <div className="rounded h-10 w-40 border border-red bg-red-l5" />
    ) : (
      <PayPalScriptProvider
        options={{
          clientId: PAYPAL_CLIENT_ID,
          commit: true,
          currency: "USD",
          enableFunding: "paylater",
          disableFunding: "card,venmo",
        }}
      >
        <div className="flex items-center gap-2">
          <Checkout orderId={orderId} source={details.source} />
        </div>
      </PayPalScriptProvider>
    )
  );
```

Reply:
applied nit: though this messes up the diff from `moved` to `deleted + new`

Reply:
We know the code was only moved, so will keep that in mind when reviewing :smile: